### PR TITLE
feat: support ReactElement for sidesheet primary and secondary action

### PR DESCRIPTION
**Pull request by finalight**

### CODE REVIEW

**Review by ninariccimarie — APPROVED** [review]
LGTM

File: src/types/componentAction.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
+import { ReactElement } from 'react';
 import { ButtonProps } from '../@next';
 
 export type ComponentAction = ButtonProps & {
-  label: string;
+  label: ReactElement;
```
Comment (ninariccimarie) [inline src/types/componentAction.ts]: can you use React.Node instead? since we want to allow any valid react child

Comment (finalight) [reply src/types/componentAction.ts]: fixed: https://github.com/glints-dev/glints-aries/pull/901/commits/bbc61f95170923680be4259e18f3b2bd2428e4f9

**Review by michac789 — APPROVED** [review]
LGTM

**Review by ninariccimarie — APPROVED** [review]
thanks LGTM

**Pull request merged**